libs/core/src/lib/segmented-button/segmented-button.component.ts

```diff
@@ -23,5 +23,5 @@ export class SegmentedButtonComponent {
 
     /** @hidden */
     @HostBinding('class.fd-segmented-button')
-    fdButtonGroupClass: boolean = true;
+    fdsegmentedButtonClass: boolean = true;
```

Comment:
maybe fdSegmentedButtonClass?

Reply:
Thats the reason for using `fd` prefix for variable names?

Reply:
variable name changed